service/src/main/java/tds/config/web/resources/TaCheckinTimeResource.java

```diff
@@ -0,0 +1,16 @@
+package tds.config.web.resources;
+
+/**
+ * A HATEOAS representation of the TA Check-In Time value.
+ */
+public class TaCheckinTimeResource {
```

Comment:
What is the purpose of this object?  Are we just making sure we have a JSON object instead of an int value returned?